distributed/core.py

```diff
@@ -673,13 +719,13 @@ def stop(self) -> None:
         self.monitor.close()
         if not (stop_listeners := self._stop_listeners()).done():
             self._ongoing_background_tasks.call_soon(
-                asyncio.wait_for(stop_listeners, timeout=None)
+                asyncio.wait_for(stop_listeners, timeout=None)  # type: ignore[arg-type]
```

Comment:
Type annotations on the `AsyncTaskGroup` are too narrow. This should be addressed in a follow-up PR.